frontend2/src/components/elements/Button.tsx

```diff
@@ -30,15 +33,17 @@ const Button: React.FC<ButtonProps> = ({
   iconName,
   fullWidth = false,
   className = "",
+  disabled,
   ...rest
 }) => {
-  const variantStyle = `${VARIANTS[variant]}  ${
+  const variantStyle = `${disabled ? DISABLED : VARIANTS[variant]} ${
```

Comment:
@acrantel Is this disabled implementation ok? Specifically my styling and disabling on load 😎

Reply:
The disabling on load is great! We should have different variants of disabled buttons b/c some places expect dark buttons and a light disabled button looks strange (e.g. login page)
![image](https://github.com/battlecode/galaxy/assets/40174697/98854b07-c8f7-4326-8986-4ec516d527c9)